GPU/Directx9/VertexShaderGeneratorDX9.cpp

```diff
@@ -232,7 +232,7 @@ void GenerateVertexShaderDX9(int prim, char *buffer, bool useHWTransform) {
 			WRITE(p, "%s", boneWeightAttrDecl[TranslateNumBones(vertTypeGetNumBoneWeights(vertType))]);
 		}
 		if (doTexture && hasTexcoord) {
-			if (doTextureProjection)
+			if (doTextureProjection && !throughmode)
```

Comment:
Your comment seems to be on an outdated line so I'm not sure exactly what you are referring to.. but the logic for whether to have a vec3 or vec2 transform seems to boil down to the same thing in both backends.. But DX9 is pretty tolerant so there might be a mistake that's ignored silently